sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/interfaces.scala

```diff
@@ -126,7 +137,9 @@ private[sql] case class AggregateExpression(
     AttributeSet(childReferences)
   }
 
-  override def toString: String = s"($aggregateFunction,mode=$mode,isDistinct=$isDistinct)"
+  override def toString: String = {
+    mode.prefix + aggregateFunction.toAggString(isDistinct)
```

Comment:
I'm wondering if it's clearer to do a pattern match here to get the prefix of mode, instead of adding the `prefix` method?